save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/fileuploader/CosvFileManagerComponent.kt

```diff
@@ -309,10 +309,9 @@ val cosvFileManagerComponent: FC<Props> = FC { _ ->
             // ===== SUBMIT BUTTONS =====
             li {
                 className = ClassName("list-group-item p-1 d-flex bg-light justify-content-center")
-                buttonBuilder("Submit", classes = "mr-1",
-                    isDisabled = selectedFiles.isEmpty() || selectedFiles.anyWithoutStatus(RawCosvFileStatus.UPLOADED) || isStreamingOperationActive) {
-                    if (window.confirm("Processed files will be removed. Do you want to continue?")) {
-                        submitCosvFiles()
+                buttonBuilder("Delete all duplicates", classes = "mr-1", isDisabled = duplicateFiles.isEmpty() || isStreamingOperationActive) {
```

Comment:
looks like `isDisabled = duplicateFiles.isEmpty()` and `availableFiles.noneWithStatus(RawCosvFileStatus.UPLOADED)` work incorrectly, since we load files by pages: it checks only first 10 files